docs/pages/Specs/nexus-prover.mdx

```diff
@@ -35,32 +35,124 @@ After receiving a Merkle proof from the initialization step with a Merkle proof
 
+One main advantage of using *Nova* is that it has a small recursion overhead, which denotes the number of steps that a prover must prove in addition to proving an invocation of $F$ [[KST22](#references)]. In particular, when used to realize an IVC scheme, Nova's recursion overhead is dominated by *two group scalar multiplications* and three hash computations. When used within a PCD, this overhead is slightly higher but still reasonable.
+
+In terms of prover costs, the most expensive part of the prover's work at the each folding step is to compute *two multi-scalar multiplications* (MSMs) of size proportional to the size of $F$. This is because the prover needs to commit to two large witness vectors at each folding step using a Pedersen commitment scheme. Despite being the most computationally expensive part of the proof generation, this computation is highly parallelizable.
+
+### Cycles of curves
+
+One difficulty that arises in the Nova implementation is that the group scalar multiplications used in the recursive step involve operations over a field that is different than the native field used by the proof system. Hence, as mentioned above, the [Nova reference implementation](https://github.com/microsoft/Nova) by Microsoft uses a *cycle of elliptic curves* to reduce the number of non-native arithmetic operations and improve its overall efficiency, This is a well-known technique introduced in [[BCTV14](#references)] to improve the efficiency of elliptic-curve-based SNARK recursions. In a $2$-_cycle_ of elliptic curves $(\mathbb{G}_1, \mathbb{G}_2)$, the size of the _base field_ of one curve equals the _scalar field_ (i.e., group order) of the other curve, and vice-versa.
+
+In a nutshell, Microsoft's reference implementation for Nova works by creating two parallel copies of the IVC chain linked together in a way that allows the recursive step of one IVC chain to be verified on the other chain using native elliptic curve arithmetic operations. As shown in [[KST22](#references)], this results in a largely symmetric scheme with a recursion overhead in the order of $20{,}000$ constraints in each of the curves. Though the Nova reference implementation can work over most cycles of curves, its current implementation uses the [`pasta` cycle](https://github.com/zcash/pasta_curves) of two curves.
```

Comment:
1. Microsoft Nova overhead is around 10000
2. They support both pasta and grumpkin cycles

Reply:
Regarding (1), I was using the number from Figure 1 in [KST22], but these numbers seem to be outdated so I will update the text. Would you have a reference for the latest numbers for the recursion overhead for Microsoft Nova?

Regarding (2), I'm adding this remark to the text.

Reply:
Added a reference for the Microsoft Nova IVC overhead